UIKitTemplate/UIKitTemplate/MenuViewController.swift

```diff
@@ -0,0 +1,7 @@
+// MenuViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Экран с выбором светофора
```

Comment:
```suggestion
/// Экран с выбором светофора
```
непонятно что за выбор светофора) ну ты понял, комментарии неоднозначные